subsys/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ add_subdirectory_ifdef(CONFIG_DFU_TARGET		dfu)
 add_subdirectory_ifdef(CONFIG_IS_SPM		spm)
 add_subdirectory_ifdef(CONFIG_TRUSTED_EXECUTION_NONSECURE nonsecure)
 add_subdirectory_ifdef(CONFIG_MPSL mpsl)
+add_subdirectory_ifdef(CONFIG_IEEE802154_NRF5 ieee802154)
```

Comment:
`CONFIG_IEEE802154_NRF5` enables the shim layer, the core radio driver is controlled by  `CONFIG_NRF_802154_RADIO_DRIVER` - are you sure we want to use the former (i. e. with Zigbee in mind)?

Reply:
Yes you are right.